### DISCUSSION TITLE
DarkTheme UI refresh?

### DISCUSSION THREAD

**Discussion by jmaraujouy** [SubtitleEdit/subtitleedit#7072] [Ideas]
I was wondering if it is possible to make a couple of changes to some UI elements in the dark theme?

I don't know... maybe changing the separators, frames and other lines to a darker color, and inverting the up/down and drop down buttons (or changing their colors).

I think, by reducing the contrast between the dark background and the lighter elements, everything looks smoother. Here's an example of what I mean:

👉🏻 Before:
![before](https://github.com/SubtitleEdit/subtitleedit/assets/117078256/fffe3442-3139-4431-a6a0-808713899d9c)

👉🏻 After:
![after](https://github.com/SubtitleEdit/subtitleedit/assets/117078256/f8bd41cf-4da8-4155-ad4b-761ddfc5f8a7)

What do you guys think?

**Comment by niksedk** [comment]
Sorry, as far as I know this is not possible with the default Winforms controls from Microsoft :(

But it looks good...

**Reply by jmaraujouy** [reply]
Oh, that's too bad. I wasn't even sure if it was possible. (I have no clue about this subject 😂).

Thanks, Nikolaj!

**Comment by jmaraujouy** [comment]
@niksedk  I was wondering... Is it possible to have buttons with rounded corners in the dark theme, just like in the default theme?

![imagen](https://github.com/SubtitleEdit/subtitleedit/assets/117078256/af82a000-e185-4227-893c-1b63ee301c91)

_Off-Topic: Also, I noticed the tabs in this section (red circle) are not showing in the default theme. (beta v570)
Although they are fully functional; if you click in the area they're suppose to be, it changes to the corresponding tab._

**Comment by niksedk** [comment]
Tabs should be back after change to default: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip

**Reply by jmaraujouy** [reply]
Tabs are back. 👍🏻 Thanks!
